configuration.k

```diff
@@ -0,0 +1,220 @@
+                    <gas>         .K          </gas>
+                    <memoryUsed>  .K          </memoryUsed>
+                    <previousGas> .K          </previousGas>
+
+                    // Those are present in both KEVM and IELE. 
+                    // Not sure we need them, let's keep them around until figure it out. 
+                    <static>    .K     </static>
+                    <callDepth>   0    </callDepth>
+                    <localCalls> .List </localCalls>
+                </callState>
+
+                // A_* (execution substate)
+                // (same as KEVM/IELE)
+                <substate>
+                    <selfDestruct> .Set  </selfDestruct>            // A_s
+                    <log>          .List </log>                     // A_l
+                    <refund>       0     </refund>                  // A_r
+                </substate>
+
+                // Immutable during a single transaction
+                // -------------------------------------
+
+                <gasPrice> 0 </gasPrice>                          // I_p
+                <origin>   0 </origin>                            // I_o
+
+                // I_H* (block information)
+                // Needed or not? It is part of KEEI, but not sure. Should discuss. 
+                <block>
+                    <hashes>           .List      </hashes>
+                    <coinbase>         0          </coinbase>         // H_c
+                    // <logsBloom>        .WordStack </logsBloom>        // H_b
+                    <difficulty>       0          </difficulty>       // H_d
+                    <number>           0          </number>           // H_i
+                    <gasLimit>         0          </gasLimit>         // H_l
+                    <timestamp>        0          </timestamp>        // H_s
+                </block>
+            </execEngine>
+            
+            // This is the part of the "Solidity VM" where we keep the available contracts. 
+            // Those are either obtained from source file (creation) or loaded from accounts (function call).
+
+            // When we parse Solidity source file we 
+            //   - desugar/preprocess modifiers, inheritance (?) etc. 
+            //   - create one <contract> structure for each contract declared in the sources
+            // (let's not worry about those for now, and just stick to simple functions)
+
+            // From here, <contract>s can be either deployed or executed. 
+            // When a contract is deployed we not only copy its <contract> cell into the <account>, 
+            // but we also copy the other contracts that were present in the same source file, 
+            // so that the deployed contract have access to them and can create and deploy them if needed. 
+            // (see the <account> cell below for more notes about this)
+
+            <contracts>
+                <Contract multiplicity = "*">
```

Comment:
this should be `type="Map"`

Reply:
OK, I will revise it accordingly.

Reply:
I just found that we define `<Contract>` as a "bag" instead of a "map". If I add `type="Mao"`, there will be errors as follows:
```[Error] Critical: failed to concatenate maps with common keys:```